config.json

```diff
@@ -604,7 +604,23 @@
         "Control flow (loops)",
         "Strings"
       ]
+    },
+    {
+      "uuid": "6ac4ad5f-a64a-4646-91c5-169d53f289f9",
+      "slug": "run-length-encoding",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 2,
+      "topics": [
+        "Control flow (conditionals)",
+        "Exception handling",
+        "Parsing",
+        "Text formatting",
+        "Regular expressions",
+        "Pattern recognition",
+        "Strings"
+      ]
```

Comment:
In this `config.json`, exercise order dictates the order in which exercises are served to the user.  Normally, we would want the exercises to progress in difficulty to the end.

Since this exercise is not the most difficult exercise in the track, I would move it to an earlier position.  I think somehwhere near 'hamming' or 'word count' would be appropriate.  I'm not fussy about exactly where.